### ISSUE TITLE
JDBC fail to get ARM native library

### ISSUE THREAD

**Issue by yonisolo** [xerial/sqlite-jdbc#57]
Hi, i come with a new problem with ARM native library call. It seems coming [from this call]
(https://github.com/xerial/sqlite-jdbc/blob/master/src/main/java/org/sqlite/core/DB.java#L161)
I use it with xowa, see this issue for the stack: gnosygnu/xowa#27
```
Error: java.lang.UnsatisfiedLinkError: org.sqlite.core.NativeDB._open(Ljava/lang/String;I)V
Stack: org.sqlite.core.NativeDB._open(Native Method)
org.sqlite.core.DB.open(DB.java:161)
org.sqlite.core.CoreConnection.open(CoreConnection.java:220)
org.sqlite.core.CoreConnection.<init>(CoreConnection.java:76)
org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:24)
org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:23)
org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:45)
org.sqlite.JDBC.createConnection(JDBC.java:114)
org.sqlite.JDBC.connect(JDBC.java:88)
java.sql.DriverManager.getConnection(DriverManager.java:664)
java.sql.DriverManager.getConnection(DriverManager.java:247)
```
this is the last post, If you have any advice... Thank you

**Comment by yonisolo** [comment]
gcc -v:
```
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/**arm-linux-gnueabihf**/4.6/lto-wrapper
Target: arm-linux-gnueabihf
Configured with: ../src/configure -v --with-pkgversion='Debian 4.6.3-14+rpi1' --with-bugurl=file:///usr/share/doc/gcc-4.6/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.6 --enable-shared --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.6 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --enable-plugin --enable-objc-gc --disable-sjlj-exceptions --with-arch=armv6 --with-fpu=vfp --with-float=hard --enable-checking=release --build=arm-linux-gnueabihf --host=arm-linux-gnueabihf --target=arm-linux-gnueabihf
Thread model: posix
gcc version 4.6.3 (Debian 4.6.3-14+rpi1)
```

**Comment by yonisolo** [comment]
My problem is solved by #58 @DavidCGG patch
